sbt-plugin/src/main/scala/com/akkaserverless/sbt/AkkaserverlessPlugin.scala

```diff
@@ -59,28 +59,33 @@ object AkkaserverlessPlugin extends AutoPlugin {
     Compile / generateUnmanaged := {
       Files.createDirectories(Paths.get((Compile / temporaryUnmanagedDirectory).value.toURI))
       // Make sure generation has happened
-      (Compile / PB.generate).value
+      val _ = (Compile / PB.generate).value
       // Then copy over any new generated unmanaged sources
       copyIfNotExist(
         Paths.get((Compile / temporaryUnmanagedDirectory).value.toURI),
         Paths.get((Compile / sourceDirectory).value.toURI).resolve("scala"))
     },
     Compile / managedSources :=
-      (Compile / managedSources).value.filter(s => !isIn(s, (Compile / temporaryUnmanagedDirectory).value)))
+      (Compile / managedSources).value.filter(s => !isIn(s, (Compile / temporaryUnmanagedDirectory).value)),
+    Compile / unmanagedSources :=
+      (Compile / generateUnmanaged).value ++ (Compile / unmanagedSources).value)
```

Comment:
Nice!

(this makes the regeneration non-optional, but that's probably fine)

Reply:
You mean, when all files are generated, you could skip that step? Probably true, but maybe no problem in practice.

Reply:
Yeah - earlier it came up that it is sometimes annoying when the code generation for user-managed  classes runs automatically - the case that came up was when the user removes the skeleton integration test (this was feedback on the java plugin). On the other hand it's nice when it's immediately generated for the first compile. For that reason we initially put the dependency in the user `build.sbt`, so they have it by default, but can easily disable it.

I now think it's fine to put it in the plugin though, not worth it to expose users to this detail for such a niche concern. We could always revisit when we get feedback.

Reply:
:+1:

Reply:
Yeah, can wait to see if this comes up with users. Can still disable by overriding the setting if needed: `Compile / generateUnmanaged := Seq.empty`.